lib/MiniDroneBtAdapter.js

```diff
@@ -238,6 +256,54 @@ class MiniDroneBtAdapter extends EventEmitter {
     }
 
     /**
+     * Convenience method for settings the drone's altitude limitation
+     * @param  {Integer} altitude the altitude in meters (2m-10m for Airborne Cargo / 2m - 25m for Mambo)
+     * @return {undefined}
+     */
+    writeMaxAltitude(altitude) {
+        const buffer = this.createBuffer(COMMAND_KEY, [MD_CLASSES.PILOTING_SETTINGS, MD_METHODS.MAX_ALTITUDE, 0x00, altitude, 0x00]);
+        this.write(COMMAND_KEY, buffer);
+        this.emit('maxAltitudeChange', altitude);
+        Logger.info(`Setting max altitude to ${altitude}m`);
+    }
+
+    /**
+     * Convenience method for setting the drone's max tilt limitation
+     * @param  {integer} tilt The max tilt from 0-100 (0 = 5° - 100 = 25°)
+     * @return {undefined}
+     */
+    writeMaxTilt(tilt) {
+        const buffer = this.createBuffer(COMMAND_KEY, [MD_CLASSES.PILOTING_SETTINGS, MD_METHODS.MAX_TILT, 0x00, tilt, 0x00]);
+        this.write(COMMAND_KEY, buffer);
+        this.emit('maxTiltChange', tilt);
+        Logger.info(`Setting max tilt to ${tilt}% (20° max)`);
+    }
+
+    /**
+     * Convenience method for setting the drone's max vertical speed limitation
+     * @param  {integer} verticalSpeed The max vertical speed from 0.5m/s - 2m/s
+     * @return {undefined}
+     */
+    writeMaxVerticalSpeed(verticalSpeed) {
+        const buffer = this.createBuffer(COMMAND_KEY, [MD_CLASSES.PILOTING_SETTINGS, MD_METHODS.MAX_VERTICAL_SPEED, 0x00, verticalSpeed, 0x00]);
+        this.write(COMMAND_KEY, buffer);
+        this.emit('maxVerticalSpeedChange', verticalSpeed);
+        Logger.info(`Setting max vertical speed to ${verticalSpeed} m/s`);
+    }
+
+    /**
+     * Convenience method for setting the drone's max rotation speed limitation
+     * @param  {integer} tilt The max rotation speed from (50°-360° for Airborne Cargo / 50° - 180° for Mambo)
+     * @return {undefined}
+     */
+    writeMaxRotationSpeed(rotationSpeed) {
+        const buffer = this.createBuffer(COMMAND_KEY, [MD_CLASSES.PILOTING_SETTINGS, MD_METHODS.MAX_ROTATION_SPEED, 0x00, rotationSpeed, 0x00]);
+        this.write(COMMAND_KEY, buffer);
+        this.emit('maxRotationSpeedChange', rotationSpeed);
+        Logger.info(`Setting max rotation speed to ${rotationSpeed} °/s`);
+    }
```

Comment:
Can we add unit tests to `MiniDroneBtAdapter-test` for these settings write methods? There's examples for other `write*` methods in there already.